app/components/primer/blankslate_component.rb

```diff
@@ -17,6 +21,14 @@ class BlankslateComponent < Primer::Component
     #     description: "Description",
     #   ) %>
     #
+    # @example 150|Loading|Add a [SpinnerComponent](https://primer.style/view-components/components/spinner) to the blankslate in place of an icon.
+    #   <%= render Primer::BlankslateComponent.new(
+    #     title: "Title",
+    #     description: "Description",
+    #   ) do |component| %>
+    #     <% component.slot(:spinner) %>
```

Comment:
Might we provide an example of an argument passed to the spinner, such as the size?